lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -711,6 +712,10 @@ data ApiWalletMigrationInfo = ApiWalletMigrationInfo
 newtype ApiWithdrawRewards = ApiWithdrawRewards Bool
     deriving (Eq, Generic, Show)
 
+newtype ApiVerificationKeyHash = ApiVerificationKeyHash
+    { verificationKeyHash :: ApiT (Hash "ScriptKey")
+    } deriving (Eq, Generic, Show)
```

Comment:
Alternatively, we could have the API return a plain bech32-encoded string on success?